pkg/virt-handler/vm.go

```diff
@@ -1386,7 +1388,7 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 			d.recorder.Event(vmi, k8sv1.EventTypeNormal, v1.Migrating.String(), "VirtualMachineInstance is migrating.")
 		}
 	} else {
-		err = client.SyncVirtualMachine(vmi)
+		err = client.SyncVirtualMachine(vmi, smbios)
```

Comment:
Perhaps it's better to pass a struct here, like `options` which will have smbios as an attribute.
Similar to `MigrateVirtualMachine` options, on line 1377 above.

Reply:
Done